plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererCodeCoverageTracker.kt

```diff
@@ -89,6 +89,10 @@ abstract class CodeWhispererCodeCoverageTracker(
                     runReadAction {
                         // also increment total tokens because accepted tokens are part of it
                         incrementTotalTokens(rangeMarker.document, originalRecommendation.length)
+                        // avoid counting CodeWhisperer inserted suggestion twice in total tokens
+                        if (rangeMarker.textRange.length in 2..49 && originalRecommendation.trim().isNotEmpty()) {
+                            incrementTotalTokens(rangeMarker.document, -rangeMarker.textRange.length)
```

Comment:
what is this for?

Reply:
When CWSPR inserts a recommendation less than 50 character, it will be counted as total characters in documentChange listener. It will also be counted in `afterAccept`. To avoid double counting the characters, this is implemented.